src/scss/form/header-form.scss

```diff
@@ -0,0 +1,147 @@
+.header-form {
+    position:fixed;
+    display:none;
+    align-items:center;
+    justify-content:center;
+    top: 0;
+    left: 0;
+    width:100%;
+    height:100%;
+    z-index: 100;
+    overflow-y: auto;
+    padding: 20px;
+    background: $button-bg-orange;
+
+    .form {
+        display: flex;
```

Comment:
You can set width: 100% by default, but with some max-width value. And you don't need to change its width on every breakpoint in this case.